common/openapi.yml

```diff
@@ -157,6 +157,35 @@ paths:
           $ref: '#/components/responses/UserError'
         '500':
           $ref: '#/components/responses/ServerError'
+  /api/batch/v1/run_sets/results:
```

Comment:
As far as we know we don't have plans to use this type of endpoint. Is it being added just in case it's useful later, or maybe I'm missing something?

Reply:
From the doc: "The callback could be one-per-workflow or could group together batches of workflows"
I can remove this API for now - we can add this later anyway.

Reply:
Ah, yeah, we certainly could do that in the future - but no plans to do it immediately.